docs/css/src/issues-list.css

```diff
@@ -1,87 +1,84 @@
 #issues-list {
-	padding: 0;
+  padding: 0;
 }
 
 #issues-list .issue {
-	border-bottom: 5px solid var(--color-bg-page);
+  border-bottom: 5px solid var(--color-bg-page);
 }
 
-	.issue .x2{
-		border-left: 10px solid;
-	}
-
-	.issue .issue-description, 
-	.issue .issue-labels {
-		padding-top: calc(var(--unit-gap) / 3); 
-		padding-bottom: calc(var(--unit-gap) / 3);
-	}
+.issue .x2 {
+  border-left: 10px solid;
+}
 
-	.issue .issue-description {
-		padding-left: var(--unit-gap);
-		font-size: 1.1em;
-	}
+.issue .issue-description,
+.issue .issue-labels {
+  padding-top: calc(var(--unit-gap) / 3);
+  padding-bottom: calc(var(--unit-gap) / 3);
+}
 
-		.issue .issue-description * {
-			line-height: 1.1em;
-		}
+.issue .issue-description {
+  padding-left: var(--unit-gap);
+  font-size: 1.1em;
+}
 
-		.issue-description .issue-number {
-			font-weight: 700;
-		}
+.issue .issue-description * {
+  line-height: 1.1em;
+}
 
-		.issue-description .issue-url {
-			text-decoration: underline;
-		}
+.issue-description .issue-number {
+  font-weight: 700;
+}
 
-		.issue-description .issue-type {
-		}
+.issue-description .issue-url {
+  text-decoration: underline;
+}
 
-		.issue-description .issue-date-open {
-			color: var(--color-default);
-			margin-top: var(--unit-gap);
-			line-height: 1em;
-			font-size: 0.8em;
-			display: inline;
-		}
+/* .issue-description .issue-type {
```

Comment:
why you out  comment this`?

Reply:
Because it's an empty selector and I didn't want to delete css selectors we're maybe using in the frontend right now.  :)